utils/docker/Dockerfile.centos.7

```diff
@@ -40,13 +40,9 @@
 #
 
 # Pull base image
-FROM centos:7.8.2003
+FROM centos:7.8.2003 AS base
```

Comment:
This doesn't appear to be valid syntax for the version of docker we run:

```
[2021-01-04T16:57:13.137Z] Error parsing reference: "centos:7.8.2003 AS base" is not a valid repository/tag: invalid reference format
```

Reply:
Could you please advise what version of docker we run?

Reply:
@JohnMalmberg would be more authoritative on that than I.

Reply:
@JohnMalmberg Could you please advise? Multi-stage build has been supported in docker since version 17.05 (released in May 2017). As i looks like centos 7.9 is using a version released in Feb 2017. Bummer.